modules/default/newsfeed/newsfeed.js

```diff
@@ -44,6 +49,10 @@ Module.register("newsfeed",{
 		return ["moment.js"];
 	},
 
+	getStyles: function() {
```

Comment:
I personally should not set this behaviour here. The custom css for users instaations should be in the custom CSS file.